pom.xml

```diff
@@ -387,6 +352,43 @@
 	</build>
 
 	<profiles>
+		<profile>
+			<id>production</id>
```

Comment:
Personally, I call this profile always 'release'

Reply:
Ok, thank you for information.